doc/getting_help.rst

```diff
@@ -1,43 +1,77 @@
 Getting help
 ============
 
+The NEST documentation has grown tremendously over the years. The following material exists:
+
+For NEST users:
+* `Read the Docs <https://nest-simulator.readthedocs.io/en/stable/>`_: This is the user 
+  documentation of NEST. Besides providing a reference, it contains various NEST tutorials 
+  and guides. Please make sure you select the documentation version appropriate for your 
+  NEST version (as determined by ``nest.version()`` in Python). The "latest" version on 
+  Read the Docs mirrors the latest version on GitHub.
+* An in-program documentation that can be accessed by ``nest.helpindex()`` or 
+  ``nest.help("term")`` in Python. This documentation describes SLI, a programming language
+  used to interact with the NEST kernel, as well as NEST models. Beware: There are naming collisions between SLI
+  and PyNEST - for example, ``nest.help("GetStatus")`` returns the documentation of the SLI
+  command, not the PyNEST function.
+
+For NEST developers:
+* `Doxygen <https://www.doxygen.nl/index.html>`_-based developer documentation, which can be built according to 
+  :doc:`these instructions <documentation_workflow/developer_documentation_workflow>`. Of course, you should 
+  develop for NEST based on the latest development version.
+* `Developer Space <https://nest.github.io/nest-simulator/>`_: This is a developer manual, 
+  explaining the development infrastructure and workflow of NEST.
+  
+Further NEST-related content and discussions can be found on our:
+
+* `GitHub page <https://github.com/nest/nest-simulator/>`_
+* `Mailing list archive <https://www.nest-simulator.org/mailinglist/hyperkitty/list/users@nest-simulator.org/>`_
+* `NEST Simulator website <https://nest-simulator.org>`_
+* `NEST Initiative webpage <https://nest-initiative.org>`_
+
+If any of the documentation is unclear, please submit an
+`Issue <https://github.com/nest/nest-simulator/issues/new?assignees=&labels=&template=documentation_improvement.md&title=>`_
+or pull request on GitHub. Please check out our :doc:`User documentation workflow <documentation_workflow/user_documentation_workflow>`.
```

Comment:
This part should be replaced with a ``:ref:`` to the "Changing the documentation" section of the "User documentation workflow".